test/faker/cat_test.exs

```diff
@@ -1,15 +1,5 @@
-defmodule CatTest do
+defmodule Faker.Cat.EnTest do
   use ExUnit.Case, async: true
 
-  test "name/0" do
-    assert is_binary(Faker.Cat.name)
-  end
-
-  test "breed/0" do
-    assert is_binary(Faker.Cat.breed)
-  end
-
-  test "registry/0" do
-    assert is_binary(Faker.Cat.registry)
-  end
-end
\ No newline at end of file
+  doctest Faker.Cat.En
```

Comment:
this one should check `Faker.Cat` doctest. Maybe it's not the bad idea to add Your `Faker.Cat.En` and other languages doctests to not multiply files for tests.